tests/php/PHPStanBearTest.py

```diff
@@ -0,0 +1,23 @@
+from bears.php.PHPStanBear import PHPStanBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+good_file = """
+<?php
+    $a = 1;
+?>
+"""
+
+
+bad_file = """
+<?php
+    $a =
+?>
+"""
+
+
+PHPStanBearTest = verify_local_bear(
```

Comment:
@satwikkansal Sorry but i need a second opinion this time.I didn't want my effort to go vain this time.As i said im not that much comfortable with `check_results`. And I haven't found any particular bear test using it.Most of the bear tests use `check_validity` and `verify_local_bear`.If i have to roll back to `check_validity` i can do that.
@Mixih Any thoughts about it ?